tests/test.rs

```diff
@@ -354,6 +354,37 @@ fn gnu_shared() {
     test.cmd(0).must_have("-shared").must_not_have("-static");
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn gnu_link_shared() {
+    use std::process::Command;
+
+    let output = Command::new("rustc").arg("-vV").output().unwrap();
```

Comment:
```suggestion
    let output = Command::new(env::var_os("RUSTC").unwrap()).arg("-vV").output().unwrap();
```

This would make sure it uses the same rustc as the cargo process that runs the tests

Reply:
Got it, in that case let's keep it as-is